src/PythHttpClient.ts

```diff
@@ -0,0 +1,138 @@
+import { AccountInfo, Commitment, Connection, PublicKey } from "@solana/web3.js";
+import { Product, PriceData, parseProductData, parsePriceData, parseBaseData, AccountType } from ".";
+
+const ONES = '1111111111111111111111111111111';
+
+interface PriceRawData {
```

Comment:
hey sorry just noticed you're using 4 spaces instead of 2. Mind running `npm run lint` before merging? that will fix the spacing issues.